### ISSUE TITLE
No graph for metrics whit name ending with "\ "

### ISSUE THREAD

**Issue by ghost** [centreon-archive/centreon-archived#1813]
---

Author Name: **Paul Ezvan** (Paul Ezvan)
Original Redmine Issue: 3890, https://forge.centreon.com/issues/3890
Original Date: 2012-11-09

---

Hi,

some of our metrics ends with "\ " (backslash and a trailing space).

The graphs of these metrics are not generated. We figured out that it was due to a RRD bug (cf https://github.com/oetiker/rrdtool-1.x/issues/331).

Indeed the generated file contains the following text error "ERROR: Unknown control code at the end of '  c-\ '".

In conclusion metrics ending with "\ " cannot be graphed by RRD, so this kind of case should be handled in Centreon (or maybe we can say it's not centreon's fault, but the rrd bug seems to not going to be resolved).
